app/serializers/post_action_type_serializer.rb

```diff
@@ -29,6 +38,10 @@ def name_key
     PostActionType.types[object.id].to_s
   end
 
+  def enabled
+    !PostActionType.disabled_flag_types[object.id]
```

Comment:
Might be nice to add an `enabled_flag_types` shortcut method to avoid the double negative here